code/game/gamemodes/clock_cult/clock_structures/ark_of_the_clockwork_justicar.dm

```diff
@@ -266,7 +266,7 @@
 					CHECK_TICK
 
 /obj/structure/destructible/clockwork/massive/celestial_gateway/attack_ghost(mob/user)
-	if(!user.client || !user.client.holder)
+	if(!user.client || !user.client.holder || (user.client && !user.client.check_rights(R_ADMIN)))
```

Comment:
the holder check should already prevent anything from happening

Reply:
_cough_ hippie mentors _cough_

Reply:
its a rippie issue, not a tg issue guys lol